Sources/Basics/JSON+Extensions.swift

```diff
@@ -0,0 +1,41 @@
+/*
+ This source file is part of the Swift.org open source project
```

Comment:
This file looks like maybe a good place to centralize the various JSON formatting options we have that need to be versioned to deployment target.

Reply:
yup will do in separate PR